fitbenchmarking/utils/fitbm_result.py

```diff
@@ -68,6 +70,33 @@ def __init__(self, options=None, problem=None, fit_status=None,
         # Used for the support pages
         self.is_best_fit = False
 
+        # Boolian that checks return true or false depending whether
+        # norm(J^T r), norm(J^T r)/norm(r) and/or norm(r) are smaller
+        # than a set tolerance
+        self._local_min = None
+
+    @property
+    def local_min(self):
+        if self.params is not None:
+            r = self.problem.eval_r(self.params)
+            min_test = np.matmul(self.problem.eval_j(self.params).T, r)
+            norm_r = np.linalg.norm(r)
+            norm_min_test = np.linalg.norm(min_test)
+            self.norm_rel = norm_min_test / norm_r
+            if norm_r <= LOCAL_MIN_TOL or norm_min_test <= LOCAL_MIN_TOL \
```

Comment:
Following on from the discussion friday, can we set two tolerances, `GRAD_TOL` and `RES_TOL`, and test `norm_r` with `RES_TOL` and the other two with `GRAD_TOL`.  We can keep the tolerance to `GRAD_TOL` as is, but tighten `RES_TOL` to, say, `1e-8`.

Reply:
Done